cmd/tink-worker/worker/worker.go

```diff
@@ -471,7 +471,7 @@ func (w *Worker) getWorkflowData(ctx context.Context, workflowID string) {
 	}
 
 	if len(res.GetData()) != 0 {
-		wfDir := filepath.Join(w.dataDir, w.workerID)
+		wfDir := filepath.Join(w.dataDir, workflowID)
```

Comment:
All other places where we are creating the filepath for the data directory, we are using workflowID.

See lines 317 and 498 in this file. Without this change, tink-worker exits pre-maturely trying to access a directory that doesn't exist (created using the workerID instead of the workflowID).